docs/current/data-model-howto.md

```diff
@@ -168,47 +167,107 @@ Boots exist as individual documents under the `boot` subdirectory. They are refe
     └── etcd-proxy.yml
 ```
 
+### Workflows
+
+Workflows exist as individual documents under the `workflows` subdirectory. They are referenced by a self-assigned ID described in the document:
+
+```
+.
+└── workflows
+    ├── clevos-accessor.yml
+    ├── k8s-master.yml
+    └── k8s-node.yml
+```
 
 ### Sites
 
-Sites are represented by individual subdirectories. One directory == one site == one managed group of machines. Each SoT can contain multiple sites. Each of these sites shares the same assets/boot/os configuration files.
+Sites are represented by individual subdirectories under the top level `sites` directory. Each `site` under the `sites` directory corresponds to a managed group
+of machines.
 
-Each site has _at least_ two documents, the specially named `env.yml` and at least one document describing an inventory of hosts. You may use YAML's triple-dash `---` separator to combine multiple inventory documents into one file.
+Each site has _at least_ two documents, the specially named `env.yml` and at least one document describing an inventory of hosts, a cluster. You may use YAML's triple-dash `---` separator to combine multiple inventory documents into one cluster file.
 
 #### Clusters
 
-Clusters are represented by individual yaml files under a site (conventional they are prefixed with `cluster-` in the file name (example: `cluster-a.yml`).
+The hosts can be grouped into multiple logical `clusters` in a `site`. Clusters are represented by individual yaml files under a site (conventional they are prefixed with `cluster-` in the file name (example: `cluster-a.yml`).
 
-One file is a grouping of hosts under that site directory. Each SoT can contain multiple sites. Each of these sites shares the same assets/boot/os configuration files.
 
 ```
 .
 └── sites
     ├── site-a
     │   ├── env.yml
-    │   └── cluster-aa.yml
-    │   └── cluster-ab.yml
+    │   ├── cluster-aa.yml
```

Comment:
I know you were just fixing the lines here but what do you think about providing more realistic names on these files? -aa -ab is probably not very typcial. I think it is worth giving users better direction here so they have a better idea how to use the cluster feature.
For example:
```
cluster-dev.yml
cluster-stage.yml
cluster-prod.yml
```

Reply:
👍